### PR TITLE
feat-added video listing page

### PR DESCRIPTION
**Pull request by prathmeshjagtap**
Added Single Video Listing Page 

Added Watch later and Like features on this Page
Functionalities of watch later and Like were checked in Previous  PR.

### CODE REVIEW

**Review by Richa-rani02 — APPROVED** [review]
code looks clean and easy understandable

**Review by Mohitkumar33 — APPROVED** [review]

File: src/App.js [hunk 1]
```diff
@@ -63,6 +64,14 @@ function App() {
 						</PrivateRoute>
 					}
 				/>
+				<Route
+					path="/video/:videoId"
+					element={
```
Comment (deepak29-git) [inline src/App.js:69]: use prettier

File: src/frontend/pages/videoListing/VideoListing.jsx [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+	const [video, setVideo] = useState("");
+
+	const { actionState, actionDispatch } = useActionContext();
+	const { dataLikes, dataWatchLater } = actionState;
+	const {
+		authState: { token },
+	} = useAuthContext();
+
+	useEffect(() => {
+		(async () => {
```
Comment (deepak29-git) [inline src/frontend/pages/videoListing/VideoListing.jsx:25]: you can make a separate function for api (just for suggestion)

**Review by deepak29-git — APPROVED** [review]
your folder structure is good

File: src/App.js (hunk 1 above)
Comment (prathmeshjagtap) [reply src/App.js:69]: yes

File: src/frontend/pages/videoListing/VideoListing.jsx (hunk 2 above)
Comment (prathmeshjagtap) [reply src/frontend/pages/videoListing/VideoListing.jsx:25]: yes i will add it

**Pull request merged**